src/store.js

```diff
@@ -2,6 +2,28 @@ import Vue from "vue";
 import Vuex from "vuex";
 
 Vue.use(Vuex);
+const groupRepeatedUnits = (templateArray = ['1fr']) => {
+  const groups = [[templateArray.shift()]];
+  for (const templateUnit of templateArray) {
+    const lastGroup = groups[groups.length - 1]
+    if (lastGroup.indexOf(templateUnit) !== -1) {
+      lastGroup.push(templateUnit);
+      continue;
```

Comment:
I get that you're trying to consolidate here, but I think a more standard `else` would be more legible. :)